### PR TITLE
Console Errors for Course Page fixed

### LINKED ISSUE #232: Remove errors from developer console window on coursepage
We need to get rid of all the errors from the developer console window from the coursepage.
Most of the errors are already removed in https://github.com/codeforcauseorg/codeforcause.org/pull/197

### PR DESCRIPTION
**Pull request by PrashubhAtri**
Fixes #232

### CODE REVIEW

File: src/components/popups/autoPopupDialog.js [hunk 1]
```diff
@@ -107,13 +107,13 @@ export default function AutoPopupDialog({ fullWidth = true, ...rest }) {
...
         <DialogContentText style={{ marginBottom: '10px' }}>
-          <Typography>
+          <Typography component={'span'}>
             Request a free counseling session with one of our mentors.
           </Typography>
         </DialogContentText>
         <ValidatorForm onSubmit={handleSubmit}>
           <TextValidator
-            autoComplete={false}
+            autoComplete=""
```
Comment (Abhishek-kumar09) [inline src/components/popups/autoPopupDialog.js:116]: You can remove this field if it was not doing anything.

File: src/components/popups/autoPopupDialog.js [hunk 2]
```diff
@@ -142,14 +142,18 @@ export default function AutoPopupDialog({ fullWidth = true, ...rest }) {
...
+                    </MenuItem>
+                  );
                 })}
               </SelectValidator>
             </Grid>
 
             <Grid item xs={10}>
               <TextValidator
-                autoComplete={false}
+                autoComplete=""
```
Comment (Abhishek-kumar09) [inline src/components/popups/autoPopupDialog.js:156]: same

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Looks really good, some minors for now.

File: src/components/popups/autoPopupDialog.js (hunk 2 above)
Comment (PrashubhAtri) [reply src/components/popups/autoPopupDialog.js]: Done :)

**Review by PrashubhAtri — COMMENTED** [review]
Completed and submitted again

**Comment by Abhishek-kumar09** [comment]
Something not working correcting: The items are shrinked.
image
image

**Comment by PrashubhAtri** [comment]
> Something not working correctly: The items are shrinked.
> image
> image

While resloving issues regarding the properties being used on Grid, I had to add "container" to it which has by default a display=flex which is causing the fault, can you please suggest a fix as I am not so familiar with material UI and there is no specific solution on the net.

**Comment by PrashubhAtri** [comment]
Please Check and review

**Comment by Abhishek-kumar09** [comment]
Thanks for your contribution @PrashubhAtri The changes are looking fine to me.

**Pull request merged**